clients/java/signalr/src/main/java/com/microsoft/signalr/HttpHubConnectionBuilder.java

```diff
@@ -22,36 +25,80 @@
         this.url = url;
     }
 
+    /**
+     * Sets the transport to be used by the {@link HubConnection}.
+     *
+     * @param transport The transport to be used.
+     * @return This instance of the HttpHubConnectionBuilder.
+     */
     public HttpHubConnectionBuilder withTransport(Transport transport) {
         this.transport = transport;
         return this;
     }
 
+    /**
+     * Sets the {@link HttpClient} to be used by the {@link HubConnection}.
+     *
+     * @param httpClient The {@link HttpClient} to be used by the {@link HubConnection}.
+     * @return This instance of the HttpHubConnectionBuilder.
+     */
     public HttpHubConnectionBuilder withHttpClient(HttpClient httpClient) {
         this.httpClient = httpClient;
         return this;
     }
 
+    /**
+     * Indicates to the {@link HubConnection} that it should skip the negotiate process.
+     * Note: This option only works with the Websockets transport and the Azure SignalR Service require the negotiate step.
```

Comment:
```suggestion
     * Note: This option only works with the Websockets transport and the Azure SignalR Service requires the negotiate step.
```

Reply:
We only really have the one transport here, I wonder if this may confuse users who aren't able to find a way to configure the transport...